jans-cedarling/scripts/parse_binding_benchmarks.py

```
@@ -0,0 +1,180 @@
+
+
+def parse_kv(text: str) -> list[dict]:
+    rows = []
+    for m in _KV_RE.finditer(text):
+        rows.append(
+            {
+                "operation": m.group(1),
+                "mean_us": float(m.group(2)),
+                "stddev_us": float(m.group(3)),
+            }
+        )
```

Ruff (0.15.12)

[warning] Use a list comprehension to create a transformed list

(PERF401)